app_test.py

```diff
@@ -121,4 +121,5 @@ def user_input_images(query):
 if __name__ == "__main__":
```

Comment:
High severity: Debug print statement may expose sensitive user data (PHI) in a medical application; risks data leakage, score: 5/10. Replace with proper logging at an appropriate level or remove before production deployment. Suggest this fix:

```suggestion
import logging
logging.debug("Query Text Input: %s", query_text_input)
```

If the print was only for temporary debugging, remove it before merging.